### PR TITLE
Tag Polynomials.jl v0.3.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaMath/Polynomials.jl](https://github.com/JuliaMath/Polynomials.jl)
Release: [v0.3.1](https://github.com/JuliaMath/Polynomials.jl/releases/tag/v0.3.1)
Travis: [Travis Build Status](https://travis-ci.org/JuliaMath/Polynomials.jl/builds/388917309)
Diff: [vs v0.3.0](https://github.com/JuliaMath/Polynomials.jl/compare/ec819d895e018365cf4baf491bfc4933bab60441...8632f056eb241b847929c23b7cc915a9726828ab)
`requires` vs v0.3.0: no changes
cc: @jverzani

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

File: Polynomials/versions/0.3.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.6
+Compat 0.48.0
```
Comment (martinholters) [inline Polynomials/versions/0.3.1/requires]: Uses `undef` now, which needs `Compat 0.59.0`.

Comment (andreasnoack) [reply Polynomials/versions/0.3.1/requires]: How did you notice? Did something break somewhere?

Comment (martinholters) [reply Polynomials/versions/0.3.1/requires]: No, nothing broke. I just saw the PR to replace `uninitialized` with `undef` and wondered whether that wouldn't require bumping the Compat version. But before I came around to checking, it had been tagged...

Comment (andreasnoack) [reply Polynomials/versions/0.3.1/requires]: Maybe easier to bump the version in the package and make a new a new release. Then the chance of this version being installed on any system is minimal.

Comment (martinholters) [reply Polynomials/versions/0.3.1/requires]: True.

Comment (jverzani) [reply Polynomials/versions/0.3.1/requires]: I just put in a PR to do this, will bump the version after testing. Thanks for catching this.

**Pull request merged**